### ISSUE TITLE
option to wipe container working directory after task exit

### ISSUE THREAD

**Issue by mlin** [chanzuckerberg/miniwdl#272]
On successful task exit, scan the output values and record all File names observed (including those nested in compound values -- task.py already has a helper function for this). Then, delete everything in the container working directory not on that list.

May be blocked on, or merge into, #271

**Comment by mlin** [comment]
Another approach: separate `miniwdl clean <rundir>` subcommand to delete all created non-output files and compress the logs

**Comment by mlin** [comment]
Easier method: make output_links hardlinks, then just rm -rf the working directory
